internal/ospkg/archlinux.go

```diff
@@ -1,4 +1,18 @@
-package packages
+// Copyright © 2021 The systemk authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
not every file needs a copyright. In fact this only needs to be done one in the License file

Reply:
Sorry, this is a thing I have for license headers, and usually, in other projects, I do have automation to put it into place. If this is something you're strongly opinionated about, I can remove all headers on files fully created by us.

Reply:
yes it's not needed, if you can please remove

Reply:
Will do.

Reply:
Done!